packages/web-components/fast-foundation/src/listbox/listbox.ts

```diff
@@ -528,6 +645,15 @@ export class DelegatesARIAListbox {
      */
     @observable
     public ariaExpanded: "true" | "false" | undefined;
+
+    /**
+     * See {@link https://w3c.github.io/aria/#listbox} for more information
+     * @public
+     * @remarks
+     * HTML Attribute: `aria-multiselectable`
+     */
+    @observable
+    public ariaMultiselectable: "true" | "false" | undefined;
```

Comment:
Should this map to [ariaMultiSelectable](https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaMultiSelectable)?